Source/Scene/Model/PointCloudStylingPipelineStage.js

```diff
@@ -125,6 +125,14 @@ PointCloudStylingPipelineStage.process = function (
     );
   }
 
+  if (pointCloudShading.backFaceCulling) {
```

Comment:
The `if` statement above this is `if(model._attenuation)`, which also comes from `pointCloudShading`. Can we make the `if` statements match each other so it's clear that their properties come from the same source? Doesn't matter which one you change, just be consistent